.yamato/config/LinuxBuildEnvironment-20170609.conf

```diff
@@ -1,6 +1,6 @@
 [LinuxBuildEnvironment-20170609]
 description=Unity Linux SDK 20170609
-directory=/home/bokken/build/output/Unity-Technologies/mono/external/buildscripts/artifacts/Stevedore/linux-sdk-20170609/linux-sdk-20170609/LinuxBuildEnvironment-20170609
+directory=/home/bokken/build/output/sources/external/buildscripts/artifacts/Stevedore/linux-sdk-20170609/linux-sdk-20170609/LinuxBuildEnvironment-20170609
```

Comment:
do you know if this is provided from yamato via env var?

Reply:
Ill find out.